From c431147e4f0c9223eb46a4174948c92b8bc93521 Mon Sep 17 00:00:00 2001
From: Kalpaj Chaudhari <daedroza@gmail.com>
Date: Sat, 18 Aug 2018 11:22:25 +0530
Subject: [PATCH 1/6] bt: osi: undef PROPERTY_VALUE_MAX

Already defined in ./bionic/libc/include/sys/system_properties.h
which causes circular errors if remove from anyone of the place.

Instead, undef first & then redefine to avoid macro-redefined errors.
Change-Id: I25f761f19e71e2859c633d8250fe146f8dd1340b
---
 osi/include/properties.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/osi/include/properties.h b/osi/include/properties.h
index b186f4f8e..8a7ea8b93 100644
--- a/osi/include/properties.h
+++ b/osi/include/properties.h
@@ -20,6 +20,8 @@
 
 #include <cstdint>
 
+#undef PROPERTY_VALUE_MAX
+
 #define PROPERTY_VALUE_MAX 92
 #define BUILD_SANITY_PROPERTY_VALUE_MAX 92
 
@@ -47,4 +49,4 @@ int32_t osi_property_get_int32(const char* key, int32_t default_value);
 //
 // returns the value of |key| coerced into a bool. If the property is not set,
 // then the |default_value| is used.
-bool osi_property_get_bool(const char* key, bool default_value);
\ No newline at end of file
+bool osi_property_get_bool(const char* key, bool default_value);
-- 
2.17.1

